### PR TITLE
Add examples for deriveBits

### PR DESCRIPTION
**Pull request by wbamberg**
Examples for the [`deriveBits()`](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/deriveBits) API. This is like [`deriveKey()`](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/deriveKey), except that `deriveKey()` also imports the bits as a `CryptoKey`, enabling you to actually use the derived key to do things like encryption.

So these examples are based on the [examples for `deriveKey()`](https://github.com/mdn/dom-examples/tree/master/web-crypto/derive-key), except they're simpler because you can't actually use the derived bits to encrypt.

I think this is the last example we need :).

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Once @april has given this a review and approved, I think we can merge it.

**Comment by wbamberg** [comment]
>  maybe you should link the mention of "deriveBits()" to the appropriate reference page on MDN, when it is available

It's a good suggestion and I don't know why I didn't do this (and BTW deriveBits is available now, I wrote the page yesterday: https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/deriveBits. But it would apply to all the examples (e.g. https://github.com/mdn/dom-examples/blob/master/web-crypto/derive-key/index.html#L14), so it would be much easier to do this as a single follow-up PR, rather than amend every open PR with this change, so can we do that?

**Comment by april** [comment]
Overall it looks fine to me, other than to say that if we're deriving a 256-bit key in `deriveKey()` then we should probably do the same in `deriveBits()`. Currently ecdh.js is deriving a 128-bit key.

**Comment by wbamberg** [comment]
Thanks again for the reviews!

> Currently ecdh.js is deriving a 128-bit key.

Again don't know I did this :). I've updated the PR.

>> it would be much easier to do this as a single follow-up PR, rather than amend every open PR with this change, so can we do that?

> Yup, absolutely.

I've filed https://github.com/mdn/dom-examples/issues/40 so we don't forget.

**Comment by chrisdavidmills** [comment]
Cool, all looks good then. Merging.

File: web-crypto/derive-bits/ecdh.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
         public: publicKey
```
Comment (nowakwp) [inline web-crypto/derive-bits/ecdh.js:8]: mdn:master

**Pull request merged**